### ISSUE TITLE
[Obsolete] MONDO:0018040 'immunodeficiency with severe reduction in serum IgG and IgA with normal/elevated IgM and normal number of B-cells'

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1667]
**Mondo term (ID and Label)**
MONDO:0018040 'immunodeficiency with severe reduction in serum IgG and IgA with normal/elevated IgM and normal number of B-cells'

**Reason for deprecation**
This is a grouping class from Orphanet that only has a single child, and is undefined. 

**Suggested term to consider**
MONDO_0015132 'immunodeficiency predominantly affecting antibody production'

_or perhaps consider:_
MONDO_0003947 'hyper-IgM syndrome'

related to https://github.com/monarch-initiative/mondo/issues/1583

**Comment by ana-rath-orphanet** [comment]
Hi Nicole,

This group shouldn't be obsoleted:
- It comes from the IUIS primary immunodeficiencies classification (Orphanet classification is totally based on it) J Clin Immunol. 2020; 40(1): 66–81.
Published online 2020 Feb 11. doi: 10.1007/s10875-020-00758-x 
- it encompasses 2 entities (and not one in Orphanet), with sub-types
![image](https://user-images.githubusercontent.com/43431008/85218896-00094180-b39f-11ea-9da3-83e6fe695998.png) 
![image](https://user-images.githubusercontent.com/43431008/85218907-1b744c80-b39f-11ea-9ec4-189dd7e72206.png)
Thank you
Ana

**Comment by cmungall** [comment]
Thanks @ana-rath-orphanet - we have held off on obsoleting this until we can gather further information. Can you comment on the issue Nicole linked to, #1583 

I hav not had time to look in detail but I don't see where the placement of mismatch repair deficiency syndrome comes from in this paper or figure?

**Comment by nicolevasilevsky** [comment]
@ana-rath-orphanet - we are going to contact the authors from the paper above to get further guidance. Thanks for your input and the link to the paper. :)

**Comment by nicolevasilevsky** [comment]
- [ ] Rename 'X-linked hyper-IgM syndrome' to hyper-IgM syndrome 1

**Comment by nicolevasilevsky** [comment]
@ana-rath-orphanet 

We consulted with one of the authors from the paper above, and it's not clear that constitutional mismatch syndrome should be classified under (MSH6), and also decided it would be more straightforward to go with the OMIM phenotypic series. For now, we decided to go ahead with obsoleting MONDO:0018040 'immunodeficiency with severe reduction in serum IgG and IgA with normal/elevated IgM and normal number of B-cells' and add an annotation to consider  'Hyper-IgM syndrome'.

Thank you for your input on this, we appreciate it. 

We'd love for you to join us on a future call! :)
